### ISSUE TITLE
User last authenticated sort is broken

### ISSUE THREAD

**Issue by mitchell852** [apache/trafficcontrol#6323]
## This Bug Report affects these Traffic Control components:
- Traffic Portal

## Current behavior:

![image](https://user-images.githubusercontent.com/251272/139922050-3a49359a-0091-4b16-8c2d-fb55bb70e259.png)

## Expected behavior:

Should sort by date and not label.

## Steps to reproduce:
in TP, navigate to https://tp.domain.tld/#!/users and show the "last authenticated" column and sort.

**Comment by ocket8888** [comment]
The issue is that for jQuery datatables, the value is coerced to a display value, then inserted into a DOM slot, then sorted on. "13 minutes ago" comes after "13 hours ago" in a lexicographic sort.

I don't know of a way to fix that with the legacy tables. However, in an ag-grid table, the display value is coerced on-demand from the actual underlying data, so the sort would be on the Date values (or `moment` instance, I think), not the particular string chosen to represent them.

tl;dr this problem can be solved by upgrading the table to use ag-grid.

**Comment by mitchell852** [comment]
> tl;dr this problem can be solved by upgrading the table to use ag-grid.

that's fine but i want to say there's a `data-sort` attribute you can use with jquery datatable columns....

**Comment by ocket8888** [comment]
Maybe, but if it's me fixing it I would say that's not worth my time to even google, when ag-grid is an all-around improvement anyway.
